### DISCUSSION TITLE
Pinia mutations color

### DISCUSSION THREAD

**Discussion by plweil** [vuejs/devtools-v6#1775] [General]
The pale yellow in the light theme is really hard to see and needs much more contrast (it's better with the dark theme). Also, the mutation "buttons" for initial mutations can be partially hidden (see screenshot). Dragging the columns (I find the columns and arrows confusing) around doesn't help. Lastly, what happened to all of the mutations details?

![Screen Shot 2022-03-04 at 3 13 07 PM](https://user-images.githubusercontent.com/1033771/156843074-e4eb9a5c-6c65-4f47-8db1-96b6716b3a6e.png)

**Comment by Akryum** [comment]
The color and event info are set by pinia. There is nothing hardcoded about this on devtools' side.

**Comment by Akryum** [comment]
You can click on the little arrows to collapse panes such as this one:
![image](https://user-images.githubusercontent.com/2798204/156852619-6b25c666-1227-42f9-a0e6-36a68dfa2bb8.png)
